crosschain/src/main/java/org/hyperledger/besu/crosschain/core/keys/CrosschainKeyManager.java

```diff
@@ -266,9 +265,7 @@ public ThresholdSignedMessage thresholdSign(final ThresholdSignedMessage message
     ThresholdSigning signer =
         new ThresholdSigning(this.p2p, this.credentials.get(this.activeKeyVersion));
     BlsPoint point = signer.sign(toBeSigned.extractArray(), message.getEncodedMessage());
-
     ThresholdSignedMessage result = message;
```

Comment:
This is a big messy - better to just have message.setSignature

Reply:
Agreed